### ISSUE TITLE
Label elements should not be keyboard focusable

### ISSUE THREAD

**Issue by joedolson** [complianz/complianz-gdpr#435]
The use of `tabindex="0"` on label elements causes a confusing double-focus when navigating inside the cookie banner using a keyboard; it also causes screen reader users to experience an extra tab stop that is unnecessary.

The added issue that the `label` element contents are hidden using `display: none` means that this focus placement will not be read by screen readers, as it effectively doesn't have any content; it will merely be an undefined focus point. The checkbox previous will be a defined focus point, as it will be recognized as a checkbox, but will also be unnamed for the same reason - because the label contents are hidden.
